ksp_plugin/part.hpp

```diff
@@ -74,12 +75,26 @@ class Part final {
   // to |PileUp::AdvanceTime| for the containing |PileUp|.  It is read and
   // cleared by |Vessel::AdvanceTime| for the containing |Vessel|.
   DiscreteTrajectory<Barycentric>& tail();
-  DiscreteTrajectory<Barycentric> const& tail() const;
 
   // True if and only if the last point of the tail is authoritative, i.e.,
-  // corresponds to a point in the psychohistory of the enclosing Part.
+  // corresponds to a point in the psychohistory of the enclosing |Vessel|.
```

Comment:
I'm confused. Should that be "a point in the history"? what is "tail()" at this point anyway?